ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1035,3 +1035,42 @@ def test_torch_matmul(
         rtol=1e-03,
         atol=1e-06,
     )
+
+
+@handle_frontend_test(
+    fn_tree="lu_factor_ex",
```

Comment:
`fn_tree` should include the module path of the function, in this case "torch.linalg.lu_factor_ex". I suppose this is the cause of the key error you are getting.